integration/kotlinx-coroutines-androidx-lifecycle/build.gradle

```diff
@@ -0,0 +1,79 @@
+import java.nio.file.Files
+import java.nio.file.NoSuchFileException
+import java.util.zip.ZipEntry
+import java.util.zip.ZipFile
+
+/*
+ * Copyright 2016-2018 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+ext.androidx_lifecycle_version = '2.0.0'
+
+repositories {
+    google()
+}
+
+def attr = Attribute.of("artifactType", String.class)
+configurations {
+    aar {
```

Comment:
Add another configuration called `aarTest` that only modifies the test classpath.

Reply:
Thanks for the hint! I had tried with `testAar`, but seems like groovy gradle messed it because of the dynamic test configurations. I have now passing tests.